core/services/commander/main.py

```diff
@@ -167,11 +166,11 @@ async def do_eeprom_update(i_know_what_i_am_doing: bool = False) -> Any:
 async def reset_settings(i_know_what_i_am_doing: bool = False) -> Any:
     check_what_i_am_doing(i_know_what_i_am_doing)
     # Be sure to not delete bootstrap to avoid going back to factory image
-    bootstrap_config_path = "/root/.config/bootstrap/startup.json"
-    temporary_location = "/tmp/bootstrap_startup.json"
-    shutil.copy2(bootstrap_config_path, temporary_location)
-    delete_everything(Path(appdirs.user_config_dir()))
-    shutil.copy2(temporary_location, bootstrap_config_path)
+    ignore = [
+        Path("/root/.config/ardupilot-manager"),
```

Comment:
Should we still delete the settings.json here? I do feel like the preferred board and MAVLink endpoints are BlueOS settings...

Reply:
It's a kind of factory reset.. so it cleans mostly everything settings-wise